services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -491,15 +491,57 @@ async def try_update_database_from_storage(
     )
     async def auto_update_database_from_storage_task(
         self, file_uuid: str, bucket_name: str, object_name: str
-    ):
+    ) -> Optional[FileMetaDataEx]:
         return await self.try_update_database_from_storage(
             file_uuid, bucket_name, object_name, silence_exception=True
         )
 
-    async def upload_link(self, user_id: str, file_uuid: str, as_presigned_link: bool):
+    async def update_metadata(
+        self, file_uuid: str, user_id: int
+    ) -> Optional[FileMetaDataEx]:
+        async with self.engine.acquire() as conn:
+            can: Optional[AccessRights] = await get_file_access_rights(
+                conn, int(user_id), file_uuid
+            )
+            if not can.write:
+                message = f"User {user_id} was not allowed to upload file {file_uuid}"
+                logger.debug(message)
```

Comment:
MINOR: instead of logging error here, you should use a middleware that logs in debug level all web.HTTPErrors .
IN addition if you might show it verbose as 
```python

# some error middleware
try:
    response = await handler(request)
 
except web.HTTPError as err:
    logger.debug( err.reason, exc_info=True, stack_info=True) 
```

Reply:
Added new middleware inside `servicelib.aiohttp` which properly logs these.